.github/workflows/main.yml

```
@@ -74,7 +44,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
         h5py-version: ["3.6", "3.7"]
```

```suggestion
        python-version: ["3.8", "3.9", "3.10"]
        h5py-version: ["3.6", "3.7"]
        include:
          - python-version: 3.11
            h5py-version: 3.7
```
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations